src/include/module/audio/audio_stream.h

```diff
@@ -16,6 +16,10 @@
 enum sof_audio_stream_state {
 	STREAM_STATE_INITIAL,			/* Initial state, hw params not configured. */
 	STREAM_STATE_READY,			/* Stream ready, hw params configured */
+	STREAM_STATE_END_OF_STREAM,		/* Detected End Of Stream */
+	STREAM_STATE_END_OF_STREAM_FLUSH,	/* Detected End Of Stream, generating silence
+						 * to flush buffers in dp modules.
+						 */
```

Comment:
Can we doxygen comment this enum (before the declaration) to make it clear this is for stream and not module/pipeline state and describe the usage/difference as I think ```state``` is getting a bit overloaded now.